client/components/NavBar.js

```diff
@@ -1,39 +1,45 @@
-import React from "react"
-import { Switch, Route, Link } from "react-router-dom"
-import PetList from "./PetList"
-import TypeList from "./TypeList"
-import RegionList from "./RegionList"
-import NewAdoptionForm from "./NewAdoptionForm"
-import AnimalSurrenderForm from "./AnimalSurrenderForm"
-
-const NavBar = props => {
-  return (
-    <div className="row column">
-      <div className="navbar">
-        <Link to="/">Home</Link>
-      </div>
-      <div className="navbar">
-        <Link to="/pets">Animal</Link>
-      </div>
-      <div className="navbar">
-        <Link to="/types">Types</Link>
-      </div>
-      <div className="navbar">
-        <Link to="/regions">Regions</Link>
-      </div>
-      <div className="navbar">
-        <Link to="/surrender">List Pet for Adoption</Link>
-      </div>
-      <Switch>
-        <Route exact path="/" component={PetList} />
-        <Route exact path="/pets" component={PetList} />
-        <Route exact path="/types" component={TypeList} />
-        <Route exact path="/regions" component={RegionList} />
-        <Route exact path="/adopt" component={NewAdoptionForm} />
-        <Route exact path="/surrender" component={AnimalSurrenderForm} />
-      </Switch>
-    </div>
-  )
-}
-
-export default NavBar
\ No newline at end of file
+import React from "react"
+import { Switch, Route, Link } from "react-router-dom"
+import PetList from "./PetList"
+import TypeList from "./TypeList"
+import RegionList from "./RegionList"
+import AnimalSurrenderForm from "./AnimalSurrenderForm"
+
+const NavBar = () => {
+  return (
+    <div id="navbar-section">
+      <div className="container">
+        <div className="logo-wrapper">
+          <img src="/images/Logo.png" alt="Endangered Animal Adoptions" />
+        </div>
+        <div className="navbar-wrapper">
+          <nav id="navbar">
+            <Link to="/pets" className="link">
+              HOME
+            </Link>
+            <Link to="#" className="link">
```

Comment:
Doesn't look like this link is functional? Might want to either update or remove - as a best practice to only merge in completed code that is functional